build.gradle

```diff
@@ -1485,7 +1485,7 @@ project(':tools') {
     testImplementation libs.junitJupiter
     testImplementation project(':clients').sourceSets.test.output
     testImplementation libs.mockitoInline // supports mocking static methods, final classes, etc.
-
+    testImplementation libs.easymock
```

Comment:
We'd better use Mockito rather than EasyMock in new PRs.

Reply:
Thank you for your opinion. I have rewritten it with mockito.